app/dashboard/templates/onepager/send2.html

```diff
@@ -101,6 +107,8 @@ <h1>{% trans "Send Tip." %}</h1>
       <br>
       <input type="checkbox" id="tos" value="1" >
       <label for="tos">{% blocktrans %}I understand &amp; agree to the <a href="https://gitcoin.co/terms">terms of service</a>.{% endblocktrans %}</label>
+      <br>

```

Comment:
Could be better to use mb-5 class and remove the break line tag, since semantically is not breaking lines here, just spacing

Reply:
updated